lib/runctl.js

```diff
@@ -150,9 +154,15 @@ function onRequest(req, callback) {
     cluster.disconnect();
 
   } else if (cmd === 'fork') {
-    var child = cluster.fork();
-    rsp.workerID = child.workerID;
-    rsp.processID = child.process.pid;
+    rsp = null;
+    cluster.fork().once('fork', function(worker) {
```

Comment:
@sam-github this is where the worker emitting a `fork` event is useful. (re: comment on strongloop/strong-cluster-control#49)